docs/examples/exampleCombined.chain.ts

```diff
@@ -117,61 +115,75 @@ async function completeProcessCombinedExamples(): Promise<void> {
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: true,
+      doRevocation: false,
+      reqUpdatedAfter: [past, past],
+      reqNonRevocationProofArr: [true, true],
+    })
+  )
 
   // without credential revocation but required dates in future => should verify
-  await completeProcessCombined({
-    blockchain,
-    expectedVerificationOutcome: true,
-    doRevocation: false,
-    reqUpdatedAfter: [future, past],
-    reqNonRevocationProofArr: [true, false],
-  })
-  await completeProcessCombined({
-    blockchain,
-    expectedVerificationOutcome: true,
-    doRevocation: false,
-    reqUpdatedAfter: [past, future],
-    reqNonRevocationProofArr: [false, true],
-  })
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: true,
+      doRevocation: false,
+      reqUpdatedAfter: [future, past],
+      reqNonRevocationProofArr: [true, false],
+    })
+  )
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: true,
+      doRevocation: false,
+      reqUpdatedAfter: [past, future],
+      reqNonRevocationProofArr: [false, true],
+    })
+  )
 
   // with revocation of 2nd credential and required date in future => should not verify
-  await completeProcessCombined({
-    blockchain,
-    expectedVerificationOutcome: false,
-    doRevocation: true,
-    reqUpdatedAfter: [future, future],
-    reqNonRevocationProofArr: [true, true],
-  })
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: false,
+      doRevocation: true,
+      reqUpdatedAfter: [future, future],
+      reqNonRevocationProofArr: [true, true],
+    })
+  )
 
   // with revocation of 2nd credential but required date in past => should verify
-  await completeProcessCombined({
-    blockchain,
-    expectedVerificationOutcome: true,
-    doRevocation: true,
-    reqUpdatedAfter: [future, past],
-    reqNonRevocationProofArr: [false, true],
-  })
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: true,
+      doRevocation: true,
+      reqUpdatedAfter: [future, past],
+      reqNonRevocationProofArr: [false, true],
+    })
+  )
 
   // with revocation (2nd) but revocation not required in verification
-  await completeProcessCombined({
-    blockchain,
-    expectedVerificationOutcome: true,
-    doRevocation: true,
-    reqUpdatedAfter: [past, past],
-    reqNonRevocationProofArr: [false, false],
-  })
+  outcomes.push(
+    await completeProcessCombined({
+      blockchain,
+      expectedVerificationOutcome: true,
+      doRevocation: true,
+      reqUpdatedAfter: [past, past],
+      reqNonRevocationProofArr: [false, false],
+    })
+  )
 
-  // disconnect from chain
-  return disconnect().finally(() => goWasmClose())
+  // check whether outcome is true fall our instances and disconnect from chain
```

Comment:
Not sure but did you mean:

```suggestion
  // check whether outcome is true for all our instances and disconnect from chain
```